pages/verify/[discordServerId]/[discordMemberId]/[customLink].tsx

```diff
@@ -41,18 +67,37 @@ const getSignatureErrorMessage = (
 const VerifyPage = ({ discordServerName, starknetNetwork }: Props) => {
   const router = useRouter();
   const { discordServerId, discordMemberId, customLink } = router.query;
-  const [account, setAccount] = useState<any>(undefined);
-  const [noStarknetWallet, setNotStarknetWallet] = useState(false);
-  const [wrongStarknetNetwork, setWrongStarknetNetwork] = useState(false);
+  const [account, setAccount] = useState<
+    StarknetAccount | StellarAccount | undefined
+  >(undefined);
+  const [noWallet, setNoWallet] = useState(false);
+  const [wrongNetwork, setWrongNetwork] = useState(false);
   const [verifyingSignature, setVerifyingSignature] = useState(false);
   const [verifiedSignature, setVerifiedSignature] = useState(false);
   const [unverifiedSignature, setUnverifiedSignature] = useState("");
   const [chainId, setChainId] = useState("");
+  const [currentChain, setCurrentChain] = useState<"starknet" | "stellar">(
+    "starknet"
+  );
+
+  useEffect(() => {
+    const fetchNetworkConfig = async () => {
+      const response = await fetch("/configs/networks.json");
```

Comment:
Please apply this comment https://github.com/nmalzieu/starky/pull/77/files#r1937107018